scylla/src/transport/load_balancing/default.rs

```diff
@@ -0,0 +1,21 @@
+use super::{LoadBalancingPolicy, Plan, Statement};
+use crate::transport::cluster::ClusterData;
+
+#[derive(Debug, Default)]
+pub struct DefaultPolicy {}
+
+impl DefaultPolicy {
```

Comment:
I wanted to remove old policies before the interface change. Removing them all would break the assumption that the `Session` has some default policy available. So in order not to break tests I chose to introduce the `DefaultPolicy` early, and incrementally improve it further in the series.